docs/user/dql/functions.rst

```diff
@@ -556,6 +583,8 @@ Argument type: INTEGER/LONG/FLOAT/DOUBLE
 
 Return type: Wider type between types of n and m if m is nonzero value. If m equals to 0, then returns NULL.
 
+Synonyms: %, MODULUS
```

Comment:
This adds a link
```suggestion
Synonyms: `MODULUS`_
```
And add a phase about `a % b` equivalency

Reply:
I have now mentioned it is a synonym to using the `Modulus Symbol (%)` and updated to add a link